algorithms/gr-dij/javascript/dijkstra_algorithm.js

```diff
@@ -0,0 +1,74 @@
+
+//created question as object
+const question = {
+    start: {A: 5, D: 8},
+    A: {B: 9, C: 3},
+    D: {C: 4, E: 6},
+    C: {B: 5, E: 2},
+    B: {end: 7},
+    E: {end: 4},
+    end: {}
+  };
+  
+  const shortestCostNode = (costs, processed) => {
+    return Object.keys(costs).reduce((lowest, node) => {
+      if (lowest === null || costs[node] < costs[lowest]) {
+        if (!processed.includes(node)) {
+          lowest = node;
+        }
+      }
+      return lowest;
+    }, null);
+  };
+  
+  // this function returns the minimum cost and path to reach end 
+  const shortestPath = (graph) => {
```

Comment:
Codacy found an issue: [Arrow function has a complexity of 7. Maximum allowed is 6.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6358206)